src/main/java/io/supertokens/webserver/api/emailpassword/UserAPI.java

```diff
@@ -111,6 +125,11 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IO
                 super.sendJsonResponse(200, result, resp);
             }
 
+        } catch (UnknownUserIdException e) {
+            JsonObject result = new JsonObject();
+            result.addProperty("status", "UNKNOWN_USER_ID_ERROR");
```

Comment:
If user has given email, then this should be UNKNOWN_EMAIL_ERROR

Reply:
Fixed